README.md

```diff
@@ -169,8 +169,13 @@ dependencies = {
 - An up-to-date `Neovim >= 0.10` installation.
 - The `git` command line utility.
 - `wget` or `curl` (if running on a UNIX system) - required for the remote `:source` command to work.
+- `make` and `unzip`
+  (if you want the install/bootstrap script to install luarocks).
 - `netrw` enabled in your Neovim configuration - enabled by default but some configurations manually disable the plugin.
-- A `lua 5.1` or `luajit` installation (for luarocks).
+- A `lua 5.1` or `luajit` installation,
+  including headers (for luarocks).
+  Note that Lua must be executable using
+  the `lua` command.
```

Comment:
I had wondered whether it needed to explicitly say that "the `lua` command must run 5.1 or luajit", as you may have someone install 5.1 but unknowingly run via 5.4.

Reply:
I'm not 100% sure, but I think that shouldn't be a problem. It's just the luarocks wrapper that has `#!/usr/bin/env lua` hardcoded.
As long as it can find Lua5.1 or luajit headers, it should be able to install Lua5.1/luajit packages, no matter which interpreter is running it.